filebeat/autodiscover/builder/hints/logs.go

```diff
@@ -139,7 +139,8 @@ func (l *logHints) CreateConfig(event bus.Event, options ...ucfg.Option) []*conf
 			kubernetes.ShouldPut(tempCfg, json, jsonOpts, l.log)
 		}
 		// Merge config template with the configs from the annotations
```

Comment:
Remove this now. Is not needed probably

Reply:
I think it is still a valid comment as it describes what is merged. And then we describe why AppendValues option is used.